### PR TITLE
Add idna-ssl to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
ref: conda-forge/aiohttp-feedstock#1

### CODE REVIEW

File: recipes/idna-ssl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (CJ-Wright) [inline recipes/idna-ssl/meta.yaml]: could this be noarch? (just put `python >= 3.5` in the build/run sections)

Comment (sodre) [reply recipes/idna-ssl/meta.yaml]: Yes, It did not occur to me that I could limit the python constraint at runtime. It makes perfect sense though.

File: recipes/idna-ssl/meta.yaml [hunk 2]
```diff
@@ -13,15 +13,15 @@ source:
...
 build:
   number: 0
+  noarch: python
   script: python -m pip install --no-deps --ignore-installed .
-  skip: True  # [not (py35 or py36)]
 
 requirements:
   build:
-    - python
+    - python >=3.5,<3.7
```
Comment (CJ-Wright) [inline recipes/idna-ssl/meta.yaml]: why less than 3.7?

Comment (sodre) [reply recipes/idna-ssl/meta.yaml]: Because of [this issue is fixed in 3.7](https://github.com/aio-libs/idna-ssl#motivation)

Comment (CJ-Wright) [reply recipes/idna-ssl/meta.yaml]: that would do it!

**Comment by CJ-Wright** [comment]
This is a noarch package and thus only needs to pass on circle.
LGTM

**Pull request merged**